File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/RowBasedCommand_View.cpp

#include "RowBasedCommand_View.h"
#include "RowBasedCommand_View.h"
#include <vector>
#include <memory>
#include "RowBasedCommand_ViewMyFlagsRow.h"
#include "RowBasedCommand_ViewMyComboBoxesRow.h"
#include "RowBasedCommand_ViewMyStandardControlsRow.h"
#include <string>

namespace commands
{
  std::vector<std::shared_ptr<RowBasedCommand_ViewMyFlagsRow>> RowBasedCommand_View::getMyFlagsListRows( ) 
  {
    return this->MyFlagsListRows;
  }
  std::vector<std::shared_ptr<RowBasedCommand_ViewMyComboBoxesRow>> RowBasedCommand_View::getMyComboBoxesTableRows( ) 
  {
    return this->MyComboBoxesTableRows;
  }
  std::vector<std::shared_ptr<RowBasedCommand_ViewMyStandardControlsRow>> RowBasedCommand_View::getMyStandardControlsTreeRows( ) 
  {
    return this->MyStandardControlsTreeRows;
  }
  void RowBasedCommand_View::loadView( ) 
  {
    
  }
  void RowBasedCommand_View::checkboxesChecked(int rowIndex, bool isChecked ) 
  {
    
  }
  void RowBasedCommand_View::freeTextComboBoxesEntrySelected(int rowIndex, std::string entryName ) 
  {
    
  }
  void RowBasedCommand_View::entryComboBoxesEntrySelected(int rowIndex, std::string entryName ) 
  {
    
  }
  void RowBasedCommand_View::textBoxesTextFilled(int rowIndex, std::string text ) 
  {
    
  }
  void RowBasedCommand_View::buttonsClicked(int rowIndex ) 
  {
    
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/RowBasedCommand_View.cs
/// <filename>
///     RowBasedCommand_View.cs
/// </filename>
using System.Collections.Generic;
using commands;

namespace commands
{
    public class RowBasedCommand_View
  {
        private System.Collections.Generic.List<commands.RowBasedCommand_ViewMyFlagsRow> MyFlagsListRows = new System.Collections.Generic.List<commands.RowBasedCommand_ViewMyFlagsRow>();
     
        public     virtual System.Collections.Generic.List<commands.RowBasedCommand_ViewMyFlagsRow> getMyFlagsListRows()
    {
      return this.MyFlagsListRows;
    }
     
        private System.Collections.Generic.List<commands.RowBasedCommand_ViewMyComboBoxesRow> MyComboBoxesTableRows = new System.Collections.Generic.List<commands.RowBasedCommand_ViewMyComboBoxesRow>();
     
        public     virtual System.Collections.Generic.List<commands.RowBasedCommand_ViewMyComboBoxesRow> getMyComboBoxesTableRows()
    {
      return this.MyComboBoxesTableRows;
    }
     
        private System.Collections.Generic.List<commands.RowBasedCommand_ViewMyStandardControlsRow> MyStandardControlsTreeRows = new System.Collections.Generic.List<commands.RowBasedCommand_ViewMyStandardControlsRow>();
     
        public     virtual System.Collections.Generic.List<commands.RowBasedCommand_ViewMyStandardControlsRow> getMyStandardControlsTreeRows()
    {
      return this.MyStandardControlsTreeRows;
    }
     
        public     virtual void loadView()
    {
      
    }
     
        public     virtual void checkboxesChecked(int rowIndex, bool isChecked)
    {
      
    }
     
        public     virtual void freeTextComboBoxesEntrySelected(int rowIndex, string entryName)
    {
      
    }
     
        public     virtual void entryComboBoxesEntrySelected(int rowIndex, string entryName)
    {
      
    }
     
        public     virtual void textBoxesTextFilled(int rowIndex, string text)
    {
      
    }
     
        public     virtual void buttonsClicked(int rowIndex)
    {
      
    }
  }
}

